front/src/common/Map/Search/MapSearch.js

```diff
@@ -14,55 +15,53 @@ export default function MapSearch(props) {
     [dispatch]
   );
 
-  const { active, toggleMapSearch } = props;
+  const { toggleMapSearch } = props;
+
   const { t } = useTranslation(['translation', 'map-search']);
-  return (
-    <div className={`map-modal${active ? ' active' : ''}`}>
-      <div className="h2">{t('translation:common.search')}</div>
 
+  const tabItems = [
+    {
+      className: 'active',
+      title: t('map-search:station'),
+      ariaSelected: 'true',
+    },
+    {
+      className: '',
+      title: t('map-search:signalbox'),
+      ariaSelected: 'false',
+    },
+    {
+      className: '',
+      title: t('map-search:signal'),
+      ariaSelected: 'false',
+    },
+  ];
+
+  return (
+    <div className="map-modal">
+      <HearderPopUp onClick={toggleMapSearch} title={t('translation:common.search')} />
```

Comment:
It would be clearer if the function name was `closeMapSearchPopUp`

Reply:
Done